static/js/compose.js

```diff
@@ -536,6 +542,34 @@ export function initialize() {
         },
     );
 
+    $("#compose_pm_mentions_user_alert").on(
+        "click",
+        ".compose_pm_mentions_user_alert_close",
+        (event) => {
+            const stream_alert_row = $(event.target).parents(".compose_pm_mentions_user_alert");
+            const stream_alert = $("#compose_pm_mentions_user_alert");
+
+            stream_alert_row.remove();
```

Comment:
The variable names here do not look correct. Some thing like `mention_alert` and `mention_alert_row` should be better.